### DISCUSSION TITLE
[Mobile] Flutter realworld app implementation

### DISCUSSION THREAD

**Discussion by realshovanshah** [realworld-apps/realworld#846] [Q&A] [answered]
I don't see a flutter based implementation of realword featured anywhere. Should I go ahead a create a example app in flutter?
Just wanted to make sure before starting it myself.

I did find a [repo](https://github.com/rhalff/realworld-flutter-app) but it isn't mentioned in realworld's docs anywhere. The repo does seem outdated though.

**Comment by geromegrignon** [comment, marked as answer]
Hey @realshovanshah, @CodeIdeal started working on one: https://github.com/CodeIdeal/realworld_flutter
I encourage you to contact them on the repository to collaborate.

**Reply by realshovanshah** [reply]
Thanks for pointing me to the repo!
I don't mean to be rude but what is the suggested action if I don't think a feature realworld repo is not actually a good realworld example?

A realworld flutter repo for contrast: https://github.com/Elias8/last_fm

**Reply by geromegrignon** [reply]
I would like to get details as I feel there might be some misunderstanding about the purpose of this project.

**Reply by realshovanshah** [reply]
The above repo (last fm) is a good example of a production level flutter project. It uses many best practices used and promoted in the flutter community which the https://github.com/CodeIdeal/realworld_flutter repo lacks.

Here's my take on a realworld flutter application.
https://github.com/realshovanshah/realworld-flutter-app

**Reply by geromegrignon** [reply]
Thanks for the details, given the potential number of implementations and related required knowledge to review them, we can't guarantee propositions to match best practices for the chosen stack.

About that part, we expect the community to provide us feedback as you just did.
I encourage you to collaborate directly with @Codeldeal on such optimization.

**Reply by realshovanshah** [reply]
Okayy, thanks for the quick response

**Comment by geromegrignon** [comment]
About the previous repo not mentioned, the creator never contacted us so we were unaware of such an implementation.
